### PR TITLE
fix: Extraction of info_adicional from BRCodes

### PR DESCRIPTION
**Pull request by renanlage**
- Extract info_adicional from static BRCodes
- Change additional_info field to additional_information in StaticPayment struct
- Remove mentions to Pix with uppercased letters

### CODE REVIEW

File: lib/ex_pix_brcode/brcode/models/brcode.ex [hunk 1]
```diff
@@ -78,7 +78,10 @@ defmodule ExPixBRCode.BRCodes.Models.BRCode do
     |> validate_inclusion(:transaction_currency, ~w(986))
     |> validate_length(:transaction_amount, max: 13)
     # Formats accept: "0", "0.10", ".10", "1.", "1", "123.99", "123456789.23"
-    |> validate_format(:transaction_amount, ~r/(^0$)|(^[0-9]+\.[0-9]{2}$)|(^[1-9]{1}([0-9])*(\.)?$)|(^\.[0-9]{2}$)/)
+    |> validate_format(
+      :transaction_amount,
+      ~r/(^0$)|(^[0-9]+\.[0-9]{2}$)|(^[1-9]{1}([0-9])*(\.)?$)|(^\.[0-9]{2}$)/
```
Comment (polvalente) [inline lib/ex_pix_brcode/brcode/models/brcode.ex:83]: This regex does not accept `"0."`. Is this intentional?

Comment (renanlage) [reply lib/ex_pix_brcode/brcode/models/brcode.ex:83]: @victorolinasc

Comment (joaogularte) [reply lib/ex_pix_brcode/brcode/models/brcode.ex:83]: Yes @polvalente. If you want to know the motivations about this regex format, I recommend to you acess this [PR#6](https://github.com/stone-co/ex-pix-brcode/pull/6).

File: lib/ex_pix_brcode/brcode/decoder.ex [hunk 2]
```diff
@@ -29,7 +30,7 @@ defmodule ExPixBRCode.BRCodes.Decoder do
   }
 
   @doc """
-  Decode input into a map with string keys with known keys for Brocade.
```
Comment (victorolinasc) [inline lib/ex_pix_brcode/brcode/decoder.ex:32]: lol

**Review by victorolinasc — APPROVED** [review]

**Pull request merged**